examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -26,17 +57,41 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import networkx as nx\n",
     "\n",
-    "from mis.shared.types import MISInstance\n",
-    "from mis.pipeline.config import SolverConfig\n",
-    "from mis import MISSolver"
+    "# Create a new networkx graph instance to be populated with DIMACS data.\n",
+    "graph = nx.Graph()\n",
+    "\n",
+    "\n",
+    "with open(\"./datasets/a265032_1tc.32.txt\", \"r\") as f:\n",
+    "    for line in f:\n",
+    "        if line.startswith(\"c\"):  # Comment line in DIMACS file.\n",
+    "            continue\n",
+    "        elif line.startswith(\"p\"):  # Problem definition, i.e. # nodes and edges.\n",
+    "            _, _, num_nodes, num_edges = line.strip().split()\n",
+    "            # Preset graph node labels as there might be isolated ones.\n",
+    "            graph.add_nodes_from(range(1, int(num_nodes) + 1))\n",
+    "\n",
+    "        elif line.startswith(\"e\"):\n",
+    "            _, node1, node2 = line.strip().split()\n",
+    "            graph.add_edge(int(node1), int(node2))\n",
+    "\n",
+    "# Let's check what the graph looks like.\n",
+    "print(graph)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solving the MIS using the classical CPLEX solver"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Solve using the configuration without quantum"
+    "Let's first solve using the MIS features and IBM's standard classical solver [Cplex](https://www.ibm.com/products/ilog-cplex-optimization-studio). "
```

Comment:
We're actually using the Networkx heuristic solver, because CPlex doesn't distribute packages for macOS.

You should probably mention that since the solver is heuristic, we're not guaranteed to have the optimal result.

Reply:
Oh, sure OK.

Reply:
Do you have a link that I can provide ?